voxblox/test/test_voxbloxpy.py

```diff
@@ -4,34 +4,36 @@
 
 import timeit
 
-import sys;
-sys.path.append('/Users/mereweth/snappy/tsdf_catkin_ws/devel/.private/voxblox/lib/')
+import sys
+sys.path.append(
+    '/Users/mereweth/snappy/tsdf_catkin_ws/devel/.private/voxblox/lib/')
```

Comment:
won't these tests fail for anyone without these hardcoded paths?

Reply:
Yeah... Maybe we should discuss moving/taking out the python bindings and tango interface, especially since we're not actually maintaining it. @mfehr @ZacharyTaylor